types/react-native/v0.69/index.d.ts

```diff
@@ -916,7 +916,7 @@ export interface LayoutRectangle {
 }
 
 // @see TextProps.onLayout
-export type LayoutChangeEvent = NativeSyntheticEvent<{ layout: LayoutRectangle }>;
+export type LayoutChangeEvent = NativeSyntheticEvent<{ layout: LayoutRectangle, target?: NodeHandle | null }>;
```

Comment:
Hmm this doesn't align with https://github.com/facebook/react-native/blob/main/Libraries/Types/CoreEventTypes.js#L75 -- am I missing something?

Reply:
Oh I see you mention the docs: https://reactnative.dev/docs/next/layoutevent, hmm is that actually true? Then we should also update the Flow types. Perhaps we can do this in another PR? Other changes look good to me!

Reply:
@lunaleaps, yeah, this is actually true :) I will update flow types and tag you, thanks!